This readme.md is outdated.Can't build from gradle.

I'm checked the  url **https://oss.sonatype.org/content/repositories/snapshots/com/google/protobuf/protobuf-gradle-plugin/**  0.7.2.SNAPSHOT is the lastest.
```
FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project 'grpc-java-demo'.
> Could not resolve all files for configuration ':classpath'.
   > Could not find com.google.protobuf:protobuf-gradle-plugin:0.8.3.
     Searched in the following locations:
         https://oss.sonatype.org/content/repositories/snapshots/com/google/protobuf/protobuf-gradle-plugin/0.8.3/protobuf-gradle-plugin-0.8.3.pom
         https://oss.sonatype.org/content/repositories/snapshots/com/google/protobuf/protobuf-gradle-plugin/0.8.3/protobuf-gradle-plugin-0.8.3.jar
     Required by:
         project :
```

Then I changed 0.8.3 to 0.7.2  got dependency error as blow:

```
FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project 'grpc-java-demo'.
> Could not resolve all files for configuration ':classpath'.
   > Could not find commons-lang:commons-lang:2.6.
     Searched in the following locations:
         https://oss.sonatype.org/content/repositories/snapshots/commons-lang/commons-lang/2.6/commons-lang-2.6.pom
         https://oss.sonatype.org/content/repositories/snapshots/commons-lang/commons-lang/2.6/commons-lang-2.6.jar
     Required by:
         project : > com.google.protobuf:protobuf-gradle-plugin:0.7.2-SNAPSHOT
   > Could not find com.google.gradle:osdetector-gradle-plugin:1.2.1.
     Searched in the following locations:
         https://oss.sonatype.org/content/repositories/snapshots/com/google/gradle/osdetector-gradle-plugin/1.2.1/osdetector-gradle-plugin-1.2.1.pom
         https://oss.sonatype.org/content/repositories/snapshots/com/google/gradle/osdetector-gradle-plugin/1.2.1/osdetector-gradle-plugin-1.2.1.jar
     Required by:
         project : > com.google.protobuf:protobuf-gradle-plugin:0.7.2-SNAPSHOT
```

--- a/README.md
+++ b/README.md
@@ -33,30 +33,17 @@ buildscript {
 }
 ```
 
-Latest changes are included in the SNAPSHOT artifact:
-```gradle
-buildscript {
-  repositories {
-    maven {
-       url 'https://oss.sonatype.org/content/repositories/snapshots/'
-    }
-  }
-  dependencies {
-    classpath 'com.google.protobuf:protobuf-gradle-plugin:0.8.3-SNAPSHOT'
-  }
-}
-```
+To try out the head version, you can download the source and build it
+with ``./gradlew install -x test`` (we skip tests here because they
+require Android SDK), then:
 
-However, the availability and freshness of the SNAPSHOT artifact are not
-guaranteed. You can instead download the source and build it with ``./gradlew
-install``, then:
 ```gradle
 buildscript {
   repositories {
     mavenLocal()
   }
   dependencies {
-    classpath 'com.google.protobuf:protobuf-gradle-plugin:0.8.3-SNAPSHOT'
+    classpath 'com.google.protobuf:protobuf-gradle-plugin:0.8.4-SNAPSHOT'
   }
 }
 ```

